src/OrbitQt/ProfilingTargetDialog.cpp

```diff
@@ -144,7 +144,8 @@ ProfilingTargetDialog::ProfilingTargetDialog(
   if (target_configuration_opt.has_value()) {
     TargetConfiguration config = std::move(target_configuration_opt.value());
     target_configuration_opt = std::nullopt;
-    SetStateMachineInitialStateFromTarget(std::move(config));
+    std::visit([this](auto target) { SetTargetAndStateMachineInitialState(std::move(target)); },
```

Comment:
Nit: `auto&& target` - Will save a move. But is also super-nit!

Reply:
I added `&&` and removed one `std::move`.